docs/gh_pages/docs/config.md

```diff
@@ -27,7 +27,7 @@ Notes:
 
 ### Classifier
 
-- `anonymizeAllEntities`: Condition to anonymize all entities in document.
+- `anonymizeSnippets`: Flag to anonymize snippet in report.
```

Comment:
nit pick : plural `sinppets`.
Also, lets mention possible values and their effect.